src/repos/FlyerRepo.ts

```diff
@@ -70,6 +70,18 @@ const getFlyersByCategorySlug = async (
     });
 };
 
+/**
+ * Looks for unique category slugs for all the Flyers
+ * @returns a list of Strings with all the category names. 
+ */
+const getAllFlyerCategories = async (
+): Promise<String[]> => {
+  return FlyerModel.collection.distinct("categorySlug")
+    .then((categories) => {
+      return categories;
+    });
```

Comment:
Do you have Prettier installed? ESLint should also point out formatting errors. Replace this function with:
```
/**
 * Looks for unique category slugs for all the Flyers
 *
 * @returns a list of Strings with all the category names.
 */
const getAllFlyerCategories = async (): Promise<string[]> => {
  return FlyerModel.collection.distinct('categorySlug').then((categories) => {
    return categories;
  });
};
```
Then, address @zachseidner1 's comment on removing `.then`.